kura/org.eclipse.kura.container.orchestration.provider/src/main/java/org/eclipse/kura/container/orchestration/provider/impl/ContainerOrchestrationServiceImpl.java

```diff
@@ -430,13 +413,14 @@ private String createContainer(ContainerConfiguration containerDescription) thro
             throw new IllegalStateException("failed to reach docker engine");
         }
 
-        if (containerDescription == null || containerDescription.getContainerImage() == null
-                || containerDescription.getContainerImageTag() == null) {
+        if (containerDescription == null || containerDescription.getImageConfiguration().getImageName() == null
```

Comment:
Can the getImageName or imageTag be null now from the imageConfiguration object?